### PR TITLE
Update scalars_and_keras.ipynb

### PR DESCRIPTION
**Pull request by rmothukuru**
Added a statement to visualize Metrics, in Tensorboard Scalars Documentation

* Motivation for features / changes : Point raised in [this issue](https://github.com/tensorflow/tensorboard/issues/4901)

* Technical description of changes : Explanation to Visualizing Metrics was missing in the documentation.

* Screenshots of UI changes

* Detailed steps to verify changes work correctly (as executed by you) : Verified it in [my Branch](https://github.com/rmothukuru/tensorboard/blob/Tensorboard_Scalars_Doc/docs/scalars_and_keras.ipynb)

* Alternate designs / implementations considered : This is just a Documentation change

### CODE REVIEW

**Review by wchargin — COMMENTED** [review]
Instead of a loud disclaimer, could we just add `metrics=['accuracy']`
to our sample model above? It seems likely that users will already be
tracking metrics that they’re interested in, so no need to draw their
attention to an **Important Note**; we can just demonstrate the right
workflow.

**Comment by rmothukuru** [comment]
> Instead of a loud disclaimer, could we just add `metrics=['accuracy']`
> to our sample model above? It seems likely that users will already be
> tracking metrics that they’re interested in, so no need to draw their
> attention to an **Important Note**; we can just demonstrate the right
> workflow.

@wchargin,
We can do that but then the Tensorboard Graph, which shows only Loss, has to be updated, and even [the Heading](https://www.tensorflow.org/tensorboard/scalars_and_keras#training_the_model_and_logging_loss), **`training_the_model_and_logging_loss`** needs to be changed. 

Please let me know how to proceed. Thanks!

**Comment by psybuzz** [comment]
To chime in, I think it is reasonable to add `metrics=['accuracy']` and update the docs accordingly, with updated headings, instructions, and screenshots as needed.
